docs/master-server.md

```diff
@@ -24,6 +26,34 @@ to complete the update. All Salt states shown should be green.
 
 ## updating the master server
 
-Whenever a change is made to the formulas or to your builder-private repository, the master server needs to be updated. It will do so every hour through a cron job, but you can always do this immediately after you want a change to be applied with:
+Whenever a change is made to the formulas, or to your `builder-private` repository, or a minion is destroyed, the master server needs to be updated. It will do so every hour through a cron job but can also be run immediately with:
+
+	./bldr master.update:master-server--prod
```

Comment:
There is potential confusion with `./bldr update:master-server--prod`, state the difference (if there is one)